### DISCUSSION TITLE
log10(Unique Fragments) vs TSS enrichment score

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1645] [General]
![image](https://user-images.githubusercontent.com/81645917/192378995-91fe367a-9761-4496-9970-81ed91a282b3.png)

Wondering how I should further refine this given the two groups in this plot, eliminate the one on the left? 

Thanks.

**Comment by rcorces** [comment]
I dont like providing advice on individual analyses as much of this is based on your own intuition of your own data. Sorry to not be of more help.

**Reply by LacquerHed** [reply]
Yes of course hard to say relative to my actual analysis and data, was more-so curious what this specifically could mean? Thanks!

**Reply by rcorces** [reply]
TSS x Frags plots can show a variety of patterns depending on the cell type, nuclei isolation method, etc. As seen in the manual, we exclude "cells" that pass TSS minimum because we dont believe these to truly be cells. But this is based on intuition rather than a heuristic cutoff.
![image](https://user-images.githubusercontent.com/10343046/192566235-fbbe71f5-d7fe-4514-a7c4-992dd1765c7a.png)

hope that helps

**Reply by LacquerHed** [reply]
Yes very helpful thank you.

**Reply by LacquerHed** [reply]
I should have posted this also earlier but I was asking because my genotype breakdown of the nuclear fragments looks like the following, so I should probably not cut those low ones in the WT sample out as they could represent some actual biology. What do you think?

![Screen Shot 2022-09-27 at 11 59 25 AM](https://user-images.githubusercontent.com/81645917/192576430-521ce384-7137-4e1b-8542-70f06a090879.png)

**Reply by rcorces** [reply]
As before, I dont want to comment on specific analyses. My reply above still stands.

**Reply by LacquerHed** [reply]
Thanks again for your help!
